src/server/scripts/EasternKingdoms/SunwellPlateau/boss_muru.cpp

```diff
@@ -448,63 +425,60 @@ class npc_void_sentinel : public CreatureScript
 public:
     npc_void_sentinel() : CreatureScript("npc_void_sentinel") { }
 
-    CreatureAI* GetAI(Creature* creature) const override
-    {
-        return new npc_void_sentinelAI(creature);
-    }
-
     struct npc_void_sentinelAI : public ScriptedAI
     {
         npc_void_sentinelAI(Creature* creature) : ScriptedAI(creature)
         {
-            Initialize();
+            _instance = me->GetInstanceScript();
+            me->SetReactState(REACT_PASSIVE);
         }
 
-        void Initialize()
+        void Reset() override
         {
-            PulseTimer = 3000;
-            VoidBlastTimer = 45000; //is this a correct timer?
-        }
+            if (Creature* muru = ObjectAccessor::GetCreature(*me, _instance->GetGuidData(DATA_MURU)))
+                muru->AI()->JustSummoned(me);
```

Comment:
I guess that's a matter of personal opinion. I for once think most of the time it's nonsense to warn in these cases and the source of the problem (or possible source) should be addressed instead of it being warned. We can (and should) consider the cases where it may have unexpected results while working on it. And personally, in this case, I really don't see anything exceptionally harmful that should be logged :)